src/main/java/com/exadel/discount/model/http/RefreshRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.exadel.discount.model.http;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class RefreshRequest {
```

Comment:
I don't think you need this class at all

Reply:
This is done.